base/server/healthcheck/pki/server/healthcheck/certs/systemcerts.py

```
@@ -39,58 +128,55 @@ def check(self):
             if cert_id == 'audit_signing' and not audit_signing_nickname:
                 continue
 
-            # Load cert trust from NSSDB
+            # Check trust flags
             with nssdb_connection(self.instance) as nssdb:
                 try:
                     cert_trust = nssdb.get_trust(
                         nickname=cert['nickname'],
                         token=cert['token']
                     )
                 except Exception as e:  # pylint: disable=broad-except
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)